src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -134,9 +149,25 @@ private JSONObject mapSpecialKeys(
       contentMetadata.put("segC", segC);
     }
 
-    int contentAssetId = properties.getInt("assetId", 0);
+    String assetIdPropertyName =
+        (settings.assetIdPropertyName != null) ? settings.assetIdPropertyName : "assetId";
+    int contentAssetId = properties.getInt(assetIdPropertyName, 0);
     contentMetadata.put("assetid", String.valueOf(contentAssetId));
 
+    String clientIdPropertyName =
+        (settings.clientIdPropertyName != null) ? settings.clientIdPropertyName : "clientId";
+    String clientId = properties.getString(clientIdPropertyName);
+    if (clientId != null && !clientId.isEmpty()) {
+      contentMetadata.put("clientId", clientId);
```

Comment:
Per the nielsen docs the key should be lowercase `clientid` in the metadata
https://engineeringportal.nielsen.com/docs/Digital_Measurement_Metadata?origin_team=T026HRLC7